index.html

```diff
@@ -1,289 +1,647 @@
-               <li class="social-media"><a class="list-style " href="#" ><img src="mediaThree.svg"     alt="" ></a></li> 
-               <li class="social-media"><a class="list-style" href="#" ><img src="mediaFour.svg"      alt="" ></a></li> 
-               <li class="social-media"><a class="list-style" href="#" ><img src="mediaFive.svg"      alt=""></a></li>
-
-            </ul>
-            
-            
+      <h1 class="intro">I’m Mariane Glad to see you!</h1>
+
+      <p class="professional-details">
+        I’m a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+
+      <h4 class="social-media-intro">LET’S CONNECT</h4>
+
+      <ul class="social-media-icon">
+        <li class="social-media">
+          <a class="list-style" href="#"><img src="tweeter.svg" alt="" /></a>
+        </li>
+        <li class="social-media">
+          <a class="list-style" href="#"
+            ><img src="Linkedin-icon.svg" alt=""
+          /></a>
+        </li>
+        <li class="social-media">
+          <a class="list-style" href="#"><img src="mediaThree.svg" alt="" /></a>
+        </li>
+        <li class="social-media">
+          <a class="list-style" href="#"><img src="mediaFour.svg" alt="" /></a>
+        </li>
+        <li class="social-media">
+          <a class="list-style" href="#"><img src="mediaFive.svg" alt="" /></a>
+        </li>
+      </ul>
     </section>
 
+    <section id="portfolio" class="card-container">
+      <!-- single card start here -->
 
-    <section class="card-container">
-
-        <!-- single card start here -->
-
-        <div class="card first-card"> 
-            <img src="cardOneImage.svg" alt="" class="card-img"> 
+      <div class="card first-card">
+        <img src="cardOneImage.svg" alt="" class="card-img" />
 
         <h2 class="project-title">Tonic</h2>
 
         <ul class="workInfo">
-            <li class="workInfoDetails first-work"><a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a> <img class="counter" src="Counter.svg" alt=""> </li>
-            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">Back End Dev</a> <img class="counter" src="Counter.svg" alt=""> </li>
-            <li class="workInfoDetails"><a class="workInfoDetailsLink" href="#">2015</a></li>
+          <li class="workInfoDetails first-work">
+            <a class="workInfoDetailsLink firstWorkLink" href="#">CANOPY</a>
+            <img class="counter" src="Counter.svg" alt="" />
+          </li>
+          <li class="workInfoDetails">
+            <a class="workInfoDetailsLink" href="#">Back End Dev</a>
+            <img class="counter" src="Counter.svg" alt="" />
+          </li>
+          <li class="workInfoDetails">
+            <a class="workInfoDetailsLink" href="#">2015</a>
```

Comment:
- [ ] Good job implementing the desktop design, but it seems some of the mobile designs were affected in the process. In this case, your project cards does not match the original design. Please ensure that both mobile and desktop designs align with the original design. 

### Your Design
![image](https://user-images.githubusercontent.com/43995830/165739441-4f309391-52d7-4f55-b6ee-ee390eb93b3f.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/165739540-15098176-5675-4735-ae5a-1ce0b6f59652.png)